docker/README.md

```diff
@@ -0,0 +1,21 @@
+# Docker for Ignite examples
+
+Usage:
+
+Pulling ignite-examples image:
+```
+docker pull pytorchignite/ignite-examples:latest
+```
+Running "/bin/sh" in the container with a mount point on "examples" from Ignite root folder:
+```
+docker run -v ${PWD}/examples:/workspace/examples -it pytorchignite/ignite-examples:latest
```

Comment:
I think we should not provide usage by mounting volumes. As you mentioned, it could create root files that could be problematic.

What do you think about `docker run --rm -it pytorchignite/ignite-examples:latest sh` ?

Reply:
Yes I completely agree, it should be safer to let the user clone the repo inside the container (git already installed). 
An explanation of the complete workflow should be great for a new user, what do you think ?

By the way, I have only tested the compliance with the docker environment with the MNIST example, I think it should be extended to the other examples.

Reply:
> An explanation of the complete workflow should be great for a new user, what do you think ?

It should be completely clear how reproduce examples in docker. It could be important to update the doc in that way too.

Present all ongoing images in docker is todo. And I think we could write a blog article to show how it works. 

What do you think ? 😃

Reply:
> As you mentioned, it could create root files that could be problematic.

Yes, sometimes it is convenient to setup docker image user as a non-root user. I have no strong opinion about that.

@fco-dv I like this README and I would replace in the begging as :
```
# Convenient docker images

In this folder we provide Dockerfiles to build docker images with PyTorch, Ignite and other convenient libraries:

- [basic](basic/Dockerfile): latest stable PyTorch, latest stable Ignite, OpenCV, Albumentations, Nvidia/Apex, etc
<!-- For example, this dockerfile I'll provide once Horovod is done -->
- [ignite-hvd](ignite-hvd/Dockerfile) : latest stable PyTorch, latest stable Ignite, Horovod
```

Reply:
> And I think we could write a blog article to show how it works.

I think docker usage should be clearly explained here and in addition we can add links on specific examples where we are using them...